scripts/core/editor3/components/spellchecker/default-spellcheckers.tsx

```diff
@@ -68,7 +69,10 @@ export function getSpellchecker(language: string): ISpellchecker {
         return {
             check: (str: string) => httpRequestJsonLocal<{errors: Array<ISpellcheckWarning>}>({
                 method: 'POST',
-                payload: {spellchecker: spellcheckerName, text: str},
+                payload: {
+                    spellchecker: spellcheckerName,
+                    text: str,
+                    ignore: ignore},
```

Comment:
Aren't ignored words stored in the database? Why send it over the wire every time spellcheck runs instead of accessing it from the back-end directly?

Reply:
Hi Tomas,

no they are not stored in database, that the case only with the `add to dictionary` feature. Ignored words are per session only and should disappear with a single refresh (see the ticket for details).

Reply:
Got it :+1: